microsoft-edge/devtools-guide-chromium/whats-new/whats-new-archive.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
in the topmost landing page file
edge-developer\microsoft-edge\developer\index.yml
change from:
```
          - text: What's new in Microsoft Edge DevTools
```
to:
```
          - text: What's New in Microsoft Edge DevTools
```

Reply:
fixed